src/atomate2/vasp/flows/ferroelectric.py

```diff
@@ -0,0 +1,123 @@
+"""Flows for calculating the polarization of a polar material."""
+
+from __future__ import annotations
+
+import logging
+from dataclasses import dataclass, field
+from pathlib import Path
+
+from jobflow import Flow, Maker
+from pymatgen.core.structure import Structure
+
+from atomate2 import SETTINGS
+from atomate2.vasp.flows.core import DoubleRelaxMaker
+from atomate2.vasp.jobs.base import BaseVaspMaker
+from atomate2.vasp.jobs.core import PolarizationMaker, RelaxMaker
+from atomate2.vasp.jobs.ferroelectric import (
+    interpolate_structures,
+    polarization_analysis,
+)
+
+__all__ = ["FerroelectricMaker"]
+
+logger = logging.getLogger(__name__)
+
+
+@dataclass
+class FerroelectricMaker(Maker):
+    """
+    Maker to calculate polarization of a polar material.
+
+    Parameters
+    ----------
+    name : str
+        Name of the flows produced by this maker.
+    nimages: int
+        Number of interpolations calculated from polar to nonpolar structures,
+        including the nonpolar.
+    """
+
+    name: str = "ferroelectric"
+    nimages: int = 9
+    symprec: float = SETTINGS.SYMPREC
+    relax: bool | tuple = False
+    bulk_relax_maker: BaseVaspMaker | None = field(
```

Comment:
Can this just be called `relax_maker` since all calculations are technically bulk.

Reply:
done